addons/website_livechat/static/src/web/channel_member_model_patch.js

```diff
@@ -4,8 +4,8 @@ import { patch } from "@web/core/utils/patch";
 /** @type {import("models").ChannelMember} */
 const channelMemberPatch = {
     getLangName() {
-        if (this.persona.is_public && this.thread.visitor?.langName) {
-            return this.thread.visitor.langName;
+        if (this.persona.is_public && this.thread.livechat_visitor_id?.lang_id.name) {
```

Comment:
We might want to `?` `persona`, `thread` and `lang_id` too in case the information is not sent in some flows?

Note: there is an incoming conflicting PR renaming `thread` to `channel_id`.